app.js

```diff
@@ -47,6 +53,13 @@ var app_manager = require('./routes/app/router-app-manager'),
     user = require('./routes/user/router-user'),
     farmer = require('./routes/resources/farmer/router-farmer');
 
+
```

Comment:
@matjames007 I've verified them. This branch still works as intended. Point taken about the code comments in future.